### ISSUE TITLE
[Bug]: Some bookmark information isn't saved/used correctly

### ISSUE THREAD

**Issue by cohenerickson** [cohenerickson/Velocity#37]
### What Happened

Information such as the icon, and relative position of a bookmark is neither saved nor used when displaying bookmarks

### Expected Behavior

```text
Bookmarks to be in the same order after page refresh and to have the correct favicon displayed
```

### Steps to Reproduce

```text
- Create 2 bookmarks from 2 different pages
- Reload the page
- Observe that bookmarks change order and don't retain favicons
```

### Other Information

```text
No further information
```

### Checklist

- [X] I have read and followed the contributing guidelines

**Comment by cohenerickson** [comment]
Bookmark favicons have been fixed, bookmark order is still broken.

**Comment by cohenerickson** [comment]
Fixed in [75f2955](https://github.com/cohenerickson/Velocity/commit/75f2955f8b31964852c42d490053bfa2ce695a6a)
